core/src/main/java/com/novoda/noplayer/internal/exoplayer/AdvertPlaybackState.java

```diff
@@ -57,6 +58,12 @@ AdPlaybackState adPlaybackState() {
         return advertBreaks;
     }
 
+    private static List<AdvertBreak> sortAdvertBreaksByStartTime(List<AdvertBreak> advertBreaks) {
+        List<AdvertBreak> sortedAdvertBreaks = new ArrayList<>(advertBreaks);
```

Comment:
We're doing this here and in the `NoPlayerAdsLoader`, do we want to make a single copy and use that?

Reply:
I guess we don't need to make another copy in the ads loader 👍